dist/sanscript.js

```diff
@@ -0,0 +1,1285 @@
+var schemes = {};
```

Comment:
This file should be ignored by Git (I think you can do `git rm --cached` or something in an interactive rebase to remove this file from the commit whence it originated).

Reply:
I'm removing the dist folder entirely.